environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python>=3.5
+  - python
```

Comment:
Hmm, I think we probably want python 3.6 or so. @redeboer Why was 3.5 introduced here? f-strings are python 3.6.

Reply:
@Leongrim introduced this minimum to fix some Conda problem in WSL, I think. There was no minimum requirement initially (the previous expertsystem required 3.3 in the `setup.py`).

Reply:
I removed this requirement because with it the expert system didn`t get installed correctly. I don´t know whyit as there though.

Reply:
Okay, I would leave it for this for now, made it into an issue #35 
Travis tests for 3.5-3.7 and I see no [literal string interpolation](https://www.python.org/dev/peps/pep-0498/) in the code base or the notebooks.

Reply:
@Leongrim #38 will cause some rebasing/merging problems